requireUp.js

```diff
@@ -0,0 +1,38 @@
+var path = require("path");
+
+/** 
+ * Attempt to require a file, recursively checking parent directories until found
+ * Similar to native `require` behavior, but doesn't check in `node_modules` folders
+ * Based on https://github.com/js-cli/node-findup-sync
+ * 
+ * @param filename {String} The name of the target file
+ * @param exts {String[]} The file extensions to look for (e.g. '.cjs')
+ * @param cwd {String} The directory to search in
+ */
+function requireUp(filename, exts, cwd) {
+  for (var i = 0; i < exts.length; i++) {
+    var filepath = path.resolve(cwd, filename) + exts[i];
```

Comment:
Fixes [this issue](https://github.com/cletusw/eslint-plugin-local-rules/pull/8#issuecomment-1152209553)